dotnet/xml/ns-Microsoft.ML.LightGBM.xml

```diff
@@ -1,6 +1,7 @@
 <Namespace Name="Microsoft.ML.LightGBM">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Namespace containing the various <a href="https://lightgbm.readthedocs.io/en/latest/">LightGBM</a> trainers.
+      This namespace is contained in the <a href="https://www.nuget.org/packages?q=Microsoft.ML.LightGBM">Microsoft.ML.LightGBM</a> nuget package. 
```

Comment:
Consider linking to nuget package directly instead of search query results

https://www.nuget.org/packages/Microsoft.ML.LightGBM/